workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -43,6 +57,11 @@ public static int RunIn(string workingDirectory, string command, params string[]
                 WorkingDirectory = workingDirectory
             };
 
+            foreach (var envVar in envVars.Keys)
```

Comment:
Why foreach over the Keys instead of iterating over the pairs?

Reply:
Good point, I'll rework this